### PR TITLE
fixed conflicts

### PR DESCRIPTION
**Pull request by w-alanna**
Code to generate random dart locations, see #4

### CODE REVIEW

File: darts.R [hunk 1]
```diff
@@ -1,12 +1,11 @@
 # Monte Carlo darts simulation to compute PI
-# Alanna Hart, Melat Ghebreselassie; November 2023
-
+# Alanna Hart, Melat Ghebreselassie, November 2023
 
 # Number of darts to throw; the more we throw, the more accurate our estimate
-
 num_darts <- 1000
-raduis <- 0.5
-points <- matrix(nrow=num_darts, ncol=2)
```
Comment (bpbond) [inline darts.R:9]: I think a `data.frame` will be easier to work down the line, so for example:
```r
points <- data.frame(x = rep(NA, num_darts), y = rep(NA, num_darts))
```

**Review by bpbond — CHANGES_REQUESTED** [review]
Hi @w-alanna thanks for starting this! See one comment below.

Also, a hint: remember that R is heavily parallelized. Is there a way to generate the point x's and y's that doesn't involve a loop?

**Review by melatg0 — CHANGES_REQUESTED** [review]
The variable raduis is spelled wrong, if changed it may be better for clarity.

**Review by melatg0 — APPROVED** [review]
Thanks Alanna!

**Pull request merged**